packages/patternfly-4/react-core/src/layouts/Flex/examples/flex.scss

```diff
@@ -0,0 +1,23 @@
+.ws-preview {
```

Comment:
Why do we need this?

Reply:
I used it because other layouts seemed to be using it. I removed it to see what would happen and it throws off some of the existing styling.